ui/js/graphs.js

```diff
@@ -522,7 +545,7 @@ perf.controller('GraphsCtrl', [
             seriesSummary.visible = partialSeries.visible;
             seriesSummary.color = availableColors.pop();
             seriesSummary.highlighted = partialSeries.highlighted;
-
+            seriesSummary.subtest_signatures = subtest_signatures;
```

Comment:
Likewise, the changes on line 537 and line 548 are not necessary because we already set the subtestSignatures property elsewhere.